vm/src/frame.rs

```diff
@@ -23,6 +23,12 @@ use crate::vm::VirtualMachine;
 use indexmap::IndexMap;
 use itertools::Itertools;
 
+#[cfg(not(target_arch = "wasm32"))]
+use crate::stdlib::signal::check_signals;
+
+#[cfg(target_arch = "wasm32")]
+fn check_signals(vm: &VirtualMachine) {}
```

Comment:
Instead of defining a stub function for wasm, make the code on line 172 also depending on configuration.